v2/guides/with-example-app/angular-golang-http.mdx

```diff
@@ -0,0 +1,143 @@
+---
+id: angular-golang-http
+title: Angular, Golang 
+hide_title: true
+hide_table_of_contents: true
+---
+
+import { GuideCondition } from "/src/components/guides/GuideCondition";
+import { GuideExampleAppInstallCommand } from "/src/components/guides/GuideExampleAppInstallCommand";
+
+import ArchitectureWithSdk from '/guides/blocks/architecture-with-sdk.mdx'
+import CustomUIHighlight from '/guides/blocks/highlight-custom-ui.mdx'
+import BackendProjectStructure from '/guides/blocks/project-structure-golang.mdx'
+import FrontendProjectStructure from '/guides/blocks/project-structure-angular.mdx'
+import SuperTokensCoreSetup from '/guides/blocks/supertokens-core-setup.mdx'
+import GuideNextSteps from '/guides/blocks/guide-next-steps.mdx'
+import GuideIntro  from '/guides/blocks/guide-intro.mdx'
+import RecipesIntro  from '/guides/blocks/recipes-intro.mdx'
+import PaidFeatureWarningMta  from '/guides/blocks/paid-feature-warning-mta.mdx'
+import PaidFeatureWarningMfa  from '/guides/blocks/paid-feature-warning-mfa.mdx'
+import RecipeMtaInfo  from '/guides/blocks/recipe-mta-info.mdx'
+import RecipeMfaInfo  from '/guides/blocks/recipe-mfa-info.mdx'
+import SessionManagement  from '/guides/blocks/session-management.mdx'
+import RecipePasswordlessInfo  from '/guides/blocks/recipe-passwordless-info.mdx'
+import RecipeSocialLoginInfo  from '/guides/blocks/recipe-social-login-info.mdx'
+import RecipeEmailPasswordInfo  from '/guides/blocks/recipe-emailpassword-info.mdx'
+import RecipeMfaConfig  from '/guides/blocks/recipe-mfa-config.mdx'
+import RecipeMtaFrontendConfig  from '/guides/blocks/recipe-mta-frontend-config.mdx'
+import RecipeMtaBackendConfig  from '/guides/blocks/recipe-mta-backend-config.mdx'
+import RecipeMtaDashboardCustomization  from '/guides/blocks/recipe-mta-dashboard-customization.mdx'
+import RecipeMfaSessionVerification  from '/guides/blocks/recipe-mfa-session-verification.mdx'
+
+
+# Guided Tutorial 
+
+<GuideIntro />
+
+## Architecture Overview
+
+First, let's get a high-level overview of how everything is supposed to work. 
+
+<ArchitectureWithSdk />
+
+## Project Structure
+
+Use our starter project to get up and running quickly. It comes with full config examples 
+based on your selection. Just run the following command to install everything:
+
+<GuideExampleAppInstallCommand />
+
+The project that you just downloaded is split into two parts:
+
+### `/backend`
+
+<BackendProjectStructure />
+
+### `/frontend`
+
+<FrontendProjectStructure />
+
+## Recipes
```

Comment:
Before this section, we need to have a section like `## Accessing the user management dashboard`. Where we walk users through visiting (localhost:3001/auth/dashboard or localhost:3000/api/auth/dashboard  in case of next js), asking them to follow the sign up instrs, and signing in. You can link to the `/docs/userdashboard/about` path also if you like.

But basically, we need to tell them that the user management dashboard is served by our backend SDK, and how to access it.